src/game_components/Lobby.js

```diff
@@ -0,0 +1,71 @@
+import { Input, Button } from "antd";
+import "antd/dist/antd.css";
+import database from "../database.js";
+
+const Lobby = (props) => {
+  const {
+    id,
```

Comment:
better to call this `gameId` in most places so you don't confuse it for a different `id`